runtime/cmake/omr_config.cmake

```diff
@@ -28,3 +28,4 @@ set(OMR_DDR OFF CACHE BOOL "")
 set(OMR_EXAMPLE OFF CACHE INTERNAL "")
 set(OMR_FVTEST OFF CACHE INTERNAL "")
 set(OMR_GC ON CACHE INTERNAL "")
+set(OMR_SEPARATE_DEBUG_INFO ON CACHE INTERNAL "")
```

Comment:
How does this take precedence over [OmrDDRSupport.cmake](https://github.com/eclipse/omr/blob/master/cmake/modules/OmrDDRSupport.cmake#L38)?

Reply:
The `INTERNAL` type implies `FORCE`. See https://cmake.org/cmake/help/latest/command/set.html

Reply:
Which one wins? The first or the second? Which is first?
Shouldn't `OMR_SEPARATE_DEBUG_INFO` be independent of DDR?

Reply:
Can you please answer my questions: Which one wins? The first or the second? Which is first?

Reply:
I believe this `set` is second, but this `set` wins because it is forced.

Reply:
yes, although order doesn't matter because this is forced, while the one in OMR is not, so regardless of order this one will win

Reply:
How is this one forced? Besides the comment, I don't see `FORCED` anywhere in this file.
Should I read the comment to mean that `INTERNAL` implies `FORCED`?

This whole discussion is moot, however: I repeat my claim that this belongs on OMR where the default should be `OMR_SEPARATE_DEBUG_INFO=ON`.